_posts/2024-02-06-opa-arrived.md

```diff
@@ -0,0 +1,110 @@
+review comments, update commits, and even a second PR, to submit a talk and
+eventually present at Trino Summit 2023 about the Open Policy Agent access
+control with Trino and their motivation to move from Apache Ranger to OPA.
+
+## OPA at Trino Summit 2023
+
+The presentation from Pablo and Sönke titled "Trino OPA authorizer - An open
+source love story" received a lot of interest from the audience at the event and
+on YouTube since then. They detailed the architectural differences of using
+Ranger and OPA. Sönke detailed the usage of OPA in the Stackable platform and
+how it enables one access control for many system. They discussed their
+collaboration on the pull request, and Pablo showed a migration path and full
+demo of OPA with Trino.
+
+{% youtube fbqqapQbAv0 %}
+
+They also made the [slide deck available for your
+reference]({{site.url}}/assets/blog/trino-summit-2023/opa-trino.pdf).
+
+Edward Morgan and Bhaarat Sharma from [Raft](https://teamraft.com/) also
+presented [Avoiding pitfalls with query federation in data
+lakehouses](https://www.youtube.com/watch?v=6KspMwCbOfI) at Trino Summit, and
+detailed their OPA usage in their Data Fabric platform. It combines Delta Lake,
+Trino, Apache Kafka, and Open Policy Agent (OPA) into a robust lakehouse data
+platform. They talked about access control in Trino overall and how important it
+is for their customers, including the US Department of Defense. Their
+presentation also included a demo of OPA with Trino.
+
+{% youtube 6KspMwCbOfI %}
+
+## OPA on the way to Trino
+
+Pablo and Sebastian continued their efforts on the [pull
+request](https://github.com/trinodb/trino/pull/19532) after Trino Summit. They
+worked successfully with Dain on the code review and necessary changes, and
+helped Manfred with the documentation.
+
+Finally, with the release of Trino 438, the [Open Policy Agent access
+control]({{site.url}}docs/current/security/opa-access-control.html) is available
+to all Trino users.
+
+The community is already taking notice with follow up pull requests for further
+improvements and blog posts such as [Enhancing Security and Observability in
+Trino with Open Policy Agent and
+OpenTelemetry](https://www.linkedin.com/pulse/enhancing-security-observability-trino-open-policy-agent-isa-inalcik-zhl9e/)
+from Isa Inalcik.
+
+## Benefits of OPA
+
+The arrival of OPA support for Trino marks an important step. OPA is a mature
+and widely used access control system. It's
+[ecosystem](https://www.openpolicyagent.org/ecosystem/) includes many
+integrations, user interfaces, development tools, and other resources. OPA it is
+a very suitable choice for a typical Trino deployment, that part of a data
+platform and pipeline consisting of many data sources, client tools, and
+applications. Trino users now have an alternative to the file-based access
+control from the Trino project itself, the effort to support your own Ranger
+integration, or the user of commercial offerings for access control.
```

Comment:
Im on mobile, so I can’t easily add a multi line comment - apologies!

what about this for replacing the sentence starting with “ OPA it is a very suitable choice for a typical …”

```
OPA is a very flexible authorization system, making it 
an ideal match for Trino. Trino deployments are often
 part of a diverse data platform, spanning a variety 
of interconnected data sources, pipelines, client 
tools and applications.
```

Reply:
Done